### PR TITLE
Fix "Python Bindings" tests for the Editor

### PR DESCRIPTION
**Pull request by jackalbe**
This fixes up the "Python Bindings" tests for the Editor
This uses the more updated Python Test Tools framework to batch the tests together and not need the Editor log files.

```
TestAutomation_BasicBatch::Editor_ComponentCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [  3%]
TestAutomation_EntityTests::Editor_LevelCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [  6%]
TestAutomation_BasicBatch::run_batched_tests[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 10%]
TestAutomation_BasicBatch::Editor_CommandLine_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 13%]
TestAutomation_BasicBatch::Editor_ComponentAssetCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 17%]
TestAutomation_BasicBatch::Editor_ComponentCommands_BuildComponentTypeNameList[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 20%]
TestAutomation_BasicBatch::Editor_ComponentPropertyCommands_enum[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 24%]
TestAutomation_BasicBatch::Editor_ComponentPropertyCommands_set_none[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 27%]
TestAutomation_BasicBatch::Editor_ComponentPropertyCommands_visibility[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 31%]
TestAutomation_BasicBatch::Editor_DisplaySettingsBus_Work[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 34%]
TestAutomation_BasicBatch::Editor_DisplaySettingsCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 37%]
TestAutomation_EntityTests::run_batched_tests[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 41%]
TestAutomation_EntityTests::Editor_ComponentPropertyCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 44%]
TestAutomation_EntityTests::Editor_EntityCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 48%]
TestAutomation_EntityTests::Editor_EntityCRUDCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 51%]
TestAutomation_EntityTests::Editor_EntitySearchCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 55%]
TestAutomation_EntityTests::Editor_EntitySelectionCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 58%]
TestAutomation_EntityTests::Editor_GameModeCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 62%]
TestAutomation_EntityTests::Editor_LevelPathsCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 65%]
TestAutomation_EntityTests::Editor_MainWindowCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 68%]
TestAutomation_EntityTests::Editor_ObjectStringRepresentation_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 72%]
TestAutomation_EditorTools::run_batched_tests[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 75%]
TestAutomation_EditorTools::Editor_PySide_Example_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 79%]
TestAutomation_EditorTools::Editor_TrackViewCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 82%]
TestAutomation_EditorTools::Editor_UtilityCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 86%]
TestAutomation_EditorTools::Editor_UtilityCommandsLegacy_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 89%]
TestAutomation_EditorTools::Editor_ViewCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 93%]
TestAutomation_EditorTools::Editor_ViewportTitleDlgCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [ 96%]
TestAutomation_EditorTools::Editor_WaitCommands_Works[windows-crash_log_watchdog0-AutomatedTesting-windows_editor] PASSED [100%]
```

### CODE REVIEW

**Review by AMZN-stankowi — APPROVED** [review]
I'm submitting this review with a "ship it" because I know a lot of these issues were already there. If you don't want to fix them now, we should just forward this feedback to the team that will maintain these tests going forward.

File: AutomatedTesting/Gem/PythonTests/EditorPythonBindings/tests/Editor_LevelComponentCommands_Works.py [hunk 1]
```diff
@@ -4,16 +4,18 @@
...
 
 def validate_component_property_apis(levelEntityComponentIdPair, componentName, componentUuid):
+    import azlmbr.legacy.general as general
+    import azlmbr.bus as bus
+    import azlmbr.editor as editor
+    from azlmbr.entity import EntityType
+
     propertyTreeOutcome = editor.EditorComponentAPIBus(bus.Broadcast, 'BuildComponentPropertyTreeEditor', levelEntityComponentIdPair)
     if not propertyTreeOutcome.IsSuccess():
         print("ERROR: BuildComponentPropertyTreeEditor failed for component with name={}, uuid={}".format(componentName, componentUuid))
```
[the remark this answers is not in the document]
Comment (jackalbe) [reply AutomatedTesting/Gem/PythonTests/EditorPythonBindings/tests/Editor_LevelComponentCommands_Works.py:21]: this is a "not True" check, so it failed

File: AutomatedTesting/Gem/PythonTests/EditorPythonBindings/TestSuite_Periodic.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+import pytest
+from ly_test_tools import LAUNCHERS
+from ly_test_tools.o3de.editor_test import EditorTestSuite, EditorBatchedTest, EditorSingleTest
+
```
[the remark this answers is not in the document]
Comment (jackalbe) [reply AutomatedTesting/Gem/PythonTests/EditorPythonBindings/TestSuite_Periodic.py:10]: https://github.com/o3de/o3de/issues/9188
https://github.com/o3de/o3de/issues/9189
https://github.com/o3de/o3de/issues/9190

File: AutomatedTesting/Gem/PythonTests/EditorPythonBindings/tests/Editor_LevelComponentCommands_Works.py [hunk 3]
```diff
@@ -4,16 +4,18 @@
...
-import sys
-import azlmbr.legacy.general as general
-import azlmbr.bus as bus
-import azlmbr.editor as editor
-from azlmbr.entity import EntityType
+def check_result(result, msg):
+    from editor_python_test_tools.utils import Report
+    if not result:
+        Report.result(msg, False)
+        raise Exception(msg + " : FAILED")
```
Comment (spham-amzn) [inline AutomatedTesting/Gem/PythonTests/EditorPythonBindings/tests/Editor_LevelComponentCommands_Works.py:11]: If these errors are from unit tests, then maybe raise a more specific exception, like ```AssertionError```?

File: AutomatedTesting/Gem/PythonTests/EditorPythonBindings/tests/Editor_ObjectManagerCommands_Works.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+def check_result(result, msg):
+    from editor_python_test_tools.utils import Report
+    if not result:
+        Report.result(msg, False)
+        raise Exception(msg + " : FAILED")
```
Comment (spham-amzn) [inline AutomatedTesting/Gem/PythonTests/EditorPythonBindings/tests/Editor_ObjectManagerCommands_Works.py:11]: I see this same function defined earlier. Maybe move the function entirely into editor_python_test_tools

**Review by spham-amzn — APPROVED** [review]

**Pull request merged**